### DISCUSSION TITLE
As Rustaceans, do you consider crab rave your theme song?

### DISCUSSION THREAD

**Discussion by OpusMag** [WayOfTheCrab/podcast#4] [Questions]
Further questions:

- What is it about Rust that is so alluring to many programmers?
- Where on the TO DO list should "learning Rust" be for new programmers? I.E. 1. Datastructures and algorithms 2. Ingraining imposter syndrome 3. Learn Rust?
- Any tips for junior devs that are a part of big organizations that just buy software they have to integrate, instead of getting to actually build things?
- Is there anything missing from Rust that you would be happy was included in an update?

**Comment by ecton** [comment]
Thank you for these questions! We ended up answering all except one of them on [episode 2](https://wayofthecrab.com/episodes/002/). For the remaining one, we discussed it before the show and didn't think we had anything interesting to add as we both have avoided working in large organizations.
